modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,111 @@
+. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
++
+[NOTE]
+It can take up to five minutes for any permissions you add to propagate.
+
+== TLS Context Configuration Prerequisites
+
+You must perform the following configuration steps prior to adding a TLS context to a secrets group:
+
+*** Have a TLS certificate with the desired Common Name set to route traffic to Runtime Fabric.
+
+.... Create a Certificate Signing Request (CSR) for a Certificate Authority (CA). To streamline this step, larger organizations have a portal/internal service that provides certificate signing from a CA or might repurpose existing certificates. If neither of the previous options are available, follow the steps required by your CA to obtain your certificate, which usually includes the following tasks:
+
+..... Submit a CSR to CA signing order. All domains/subdomains for serving traffic must be included in the Subject Alternative Name (SAN).
+..... Prove domain ownership (DV certificates) or complete the necessary background checks for the certificate level.
+..... Download the certificate and CA path (signing path) from your CA.
+.... Review domains associated with the  (SAN) in your certificate. Many clients, including most browsers, only return a lock icon, which indicates a verified TLS connection, if the URL matches a domain in the SAN field of the certificate. 
+.... Be prepared to add or update the DNS routing record to send traffic to the external load balancer IP address or addresses that front the Runtime Fabric instance. This is the final step needed to cut in or cut over traffic.
+
+*** Be able to create or configure an external TCP load balancer to manage traffic to applications running in Runtime Fabric:
+
+.... Gather all IP addresses of controller or dedicated nodes (depending on the configuration you choose during installation) that can be reached by the external TCP load balancer. These IP addresses may not be known until you complete the Runtime Fabric installation. For smaller traffic loads, controller nodes can be shared for cluster administration and inbound traffic TLS termination. For larger traffic loads, use dedicated nodes. The controller and dedicated nodes must have IP addresses that cn be reached (are routable and allowed at applicable firewalls) by the external load balancer. Otherwise, traffic cannot be routed from the load balancer to Runtime Fabric. 
+.... Know the domain or domains to be used for routing traffic to your applications.
+.... Understand that you need to configure an external load balancer to proxy the TCP protocol. The load balancer does not terminate TLS but does allow Runtime Fabric to do so.
+
+*** Be able to configure the DNS settings to associate domains that are signed in the TLS certificate to the IP address or addess(es) served by the external load balancer.
```

Comment:
These lines are true for anyone enabling inbound traffic with a certificate, even if that certificate doesn't come from Secrets Manager.

Reply:
Thank you - moved this to the other topic - please see if it looks OK.